src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "0.5.144"
+VERSION = "0.5.145"
```

Comment:
This version number has been used/release, please choose a new one.

Reply:
changed to 0.5.146